apps/admin-page/src/app/app.component.html

```diff
@@ -36,7 +37,7 @@
         data-cy="sidebar"
     >
         <div class="sidebar-header" data-cy="sidebar-header">
-            <mat-icon aria-hidden="false" data-cy="sidebar-user-icon">face</mat-icon>
+            <mat-icon aria-hidden="false" data-cy="sidebar-user-icon" routerLink="account">face</mat-icon>
```

Comment:
The cursor doesn't change when hovering over the icon, making it harder to figure out that it will redirect to the account page